runner/app/live/infer.py

```diff
@@ -21,6 +21,34 @@
 from streamer.protocol.zeromq import ZeroMQProtocol
 
 
+def asyncio_exception_handler(loop, context):
+    """
+    Handles unhandled exceptions in asyncio tasks, logging the error and terminating the application.
+    """
+    exception = context.get('exception')
+    logging.error(f"Caught unhandled exception in asyncio task: {exception}")
```

Comment:
```python
exception = context.get('exception')
logging.error(f"Caught unhandled exception in asyncio task", exc_info=exception)
os.exit(1)
```

Reply:
Oh nice, didn't know `exc_info` could get the exception directly